src/main/java/com/passbook/sparkeighteen/controller/UserController.java

```diff
@@ -24,17 +26,21 @@ public UserController(final UserService userService) {
         this.userService = userService;
     }
 
-    @ApiOperation("sign-up the user to passbook")
+    @ApiOperation("Sign-Up the user to passbook")
     @PostMapping("/sign-up")
-    public ResponseEntity<SignUpResponse> signUp(@Valid @RequestBody final SignUpRequest signUpRequest) throws Exception {
-        return ResponseEntity.ok(userService.signUp(signUpRequest));
+    public ResponseEntity<SignUpResponse> signUp(@Valid @RequestBody final SignUpRequest request) throws Exception {
+        SignUpResponse response = userService.signUp(request);
+        if (response.getUserID() != null)
```

Comment:
This logic has to be in service, return response based on condition from service. In Controller you just have to wrap response in responseEntity